docs/installation.rst

```diff
@@ -11,7 +11,46 @@ provided with pulpcore.
 Install ``pulp_rpm``
 --------------------
 
-Users should install from **either** PyPI or source.
+Users should install from **either** PyPI or source or use ansible-pulp installer.
+
+Install by Ansible-pulp
+***********************
+
+With use of ``ansible-pulp`` installer you need to download supportive
+role before you run installer.
```

Comment:
s/installer/the installer/

Reply:
I think here its together "the ansible-pulp installer'

Reply:
Yup you're right